### ISSUE TITLE
Using backbone-relational in node doesn't work anymore

### ISSUE THREAD

**Issue by tinganho** [PaulUithol/Backbone-relational#482]
Hi @PaulUithol ,

I use `backbone-relational` heavily on nodejs. In the current release. The export statement in the factory doesn't export Backbone.

**Comment by cj3kim** [comment]
Just chiming in. IIRC, require('backbone') returns a singleton object. It would be much cleaner to modify/kickoff Backbone with Backbone-Relational by doing the following:

    Backbone = require('backbone-relational')(Backbone);

I understand require('backbone-relational') internally utilizes the same reference, but I think it's a poor pattern for backbone-relational to internally expect an external variable to be in its outer scope. It's much cleaner to utilize dependency injections.

**Comment by PaulUithol** [comment]
Soo.. I'm not much of a node user, so sorry if I don't understand correctly, but is there anything we can change in Backbone-relational to make this easier to work with?

Shouldn't `factory( exports, require( 'backbone' ), require( 'underscore' ) );` take care of importing Backbone, as far as we're concerned?

**Comment by denisjarus** [comment]
The point is:
```
require('backbone-relational') === undefined
```
and
```
var Backbone = require('backbone');
require('backbone-relational');
```
gives you just Backbone without relational, so the entire thing is totally unusable in node.js

**Comment by ilovett** [comment]
This seems to be working

```
var Backbone = require('backbone');
require('backbone-relational');
console.log(typeof Backbone.RelationalModel); // outputs [Function]
```

**Comment by bpatram** [comment]
This issue should be fixed with: #559
